test/integration/webhook/validating_webhook_suite_test.go

```diff
@@ -97,4 +116,12 @@ var _ = AfterSuite(func() {
 	if securityGroupID != "" {
 		Expect(frameWork.EC2Manager.DeleteSecurityGroup(ctx, securityGroupID)).To(Succeed())
 	}
+
+	By("Removing the patch on aws-node cluster role")
+	if updatedClusterRole != nil {
+		updatedClusterRole.Rules = updatedClusterRole.Rules[:len(updatedClusterRole.Rules)-1]
```

Comment:
Could this be a potential problem? I am not sure if the new permission can always be the last one when server returns. Although this will be the last test, I think we should still be resilient. Can you make it specifically delete the new permission on node? Thanks.